addon/components/base/bs-modal/dialog.js

```diff
@@ -201,6 +194,11 @@ export default Component.extend({
     // directly to the element
     this.element.onclick = bind(this, this._click);
     this.getOrSetTitleId();
+    this.updateStyles();
```

Comment:
Could not use `didReceiveAttrs` cause that one is executed before element is insert into DOM.